src/pal/inc/rt/objidl.h

```diff
@@ -101,12 +101,12 @@ enum tagSTREAM_SEEK
 	STREAM_SEEK_END	= 2
     } 	STREAM_SEEK;
 
-typedef 
-enum tagLOCKTYPE
-    {	LOCK_WRITE	= 1,
-	LOCK_EXCLUSIVE	= 2,
-	LOCK_ONLYONCE	= 4
-    } 	LOCKTYPE;
+//typedef 
+//enum tagLOCKTYPE
+//    {	LOCK_WRITE	= 1,
+//	LOCK_EXCLUSIVE	= 2,
+//	LOCK_ONLYONCE	= 4
+//    } 	LOCKTYPE;
```

Comment:
You can safely remove these lines here; no need to keep unused code commented-out.  (Version control allows this to be easily found and reverted if needed.)

Reply:
Yes, feel free to delete the whole thing.

Reply:
Done.